nixos/modules/installer/tools/tools.nix

```diff
@@ -170,7 +170,8 @@ in
         # List packages installed in system profile. To search, run:
         # \$ nix search wget
         # environment.systemPackages = with pkgs; [
-        #   wget vim
+        #   nano vim # don't forget to add an editor to edit configuration.nix!
```

Comment:
Note that nano is always installed.

Reply:
Where is this set in nixos? I could not find it.

Reply:
https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/config/system-path.nix#L45

Reply:
> Note that nano is always installed.

> Where is this set in nixos? I could not find it.

I'm not entirely sure, but I think that was here:
https://github.com/NixOS/nixpkgs/blob/e8c4b71d31ef12f2b593ff59d01b222c307e12bc/nixos/modules/config/system-path.nix#L10-L32

However, `nano` has since (in #97565) been moved to `defaultPackages` that users _can_ remove if they so choose.

Reply:
follow up PR: https://github.com/NixOS/nixpkgs/pull/120023